src/SiteExtensions/Sdk/SiteExtension.targets

```
@@ -0,0 +1,83 @@
+<!--
+  This file is automatically imported by NuGet into a user's project
+  when it targets a single framework, or in classic (pre 2017) csproj projects.
+-->
+
+<Project>
+
+  <Target Name="GenerateHostingStartupDeps" Condition="'@(HostingStartupPackageReference->Count())' != '0'" BeforeTargets="_GetPackageFiles" >
+
+    <PropertyGroup>
+      <_TemplatesDirectory>$(MSBuildThisFileDirectory)..\content\</_TemplatesDirectory>
+      <_DepsOutputDirectory>$(RepositoryRoot)\obj\lse\</_DepsOutputDirectory>
```

```suggestion
      <_DepsOutputDirectory>$(IntermediateOutputPath)lse\</_DepsOutputDirectory>
```
Let's try using idiomatic MSBuild output locations. You can set this targets file to align with the project's obj directory. 
IIRC, MAX_PATH was issue due to the deep directory nesting of the store, so I'd also recommend adding this content to `src/SiteExtensions/Directory.Build.props`:
```xml
<Project>
  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />

  <PropertyGroup>
    <BaseIntermediateOutputPath>$(RepositoryRoot)obj\$(MSBuildProjectName)\</BaseIntermediateOutputPath>
    <BaseOutputPath>$(RepositoryRoot)bin\$(MSBuildProjectName)\</BaseOutputPath>
  </PropertyGroup>
</Project>
```